doc/examples/plot_matching.py

```diff
@@ -120,7 +146,20 @@ def match_corner(coord, window_ext=5):
 print(model_robust.scale, model_robust.translation, model_robust.rotation)
 
 # visualize correspondence
-fig, ax = plt.subplots(nrows=2, ncols=1)
+fig, ax = plt.subplots(nrows=4, ncols=1)
+
+def warp_and_combine(img1,img2,xform):
+    img1_warped = warp(img1,xform)
```

Comment:
I didn't look carefully, but you might need to swap the 0-th and 1-th dimension as the warp function expects y, x and the affine transform might return x, y.